Сетка страниц и крупных блоков на флексах

Tigo929 opened the pull request:
Добавил всю графику, которая должна располагаться в разметку.

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/38/user/2388837/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/2388837-sedona-38/3/

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/38/user/2388837/tasks/8)

---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/38/user/2388837/tasks/10)

Tigo929 committed:
Добавляет графику

Добавил всю графику, которая должна располагаться в разметку.

Tigo929 committed:
Добавляет контеное  изображение и редактирует код

Добавил изображение, а также использовал методологию бэм

eepodoprigora commented:
Привет, 
1. Посмотрела в фигме у иконок сердечка и поиска нет состояний, поэтому давай перенесем их в css как фон к ссылке. services-item картинки тоже будут без состояний, так что в css . И вот эти звездочки тоже декоративные в CSS, вместе img в разметке нужно сделать количество span равное количеству звезд,  и дать каждому по дной звезде фоном.

Отличная работа!

Tigo929 committed:
CСОздаёт начальную базовую верстку

АААА

Tigo929 committed:
Добавляет базовую стилизацию для indexa

=)

Tigo929 committed:
Добавляет базовую стилизацию

Сделал базовую  стилизацию, но возникли следующие трудности:
1) с классом visually-hidden- не понимаю как скрыть тег, но чтобы изображение отображалось( heder: сердечко и избранное; хлебные,крошки).
2) Не смог найти решение для того, чтобы  отобразить звезда через css.
В принципе по базовой стилизации всё остальное доделал.

Tigo929 committed:
Merge branch 'htmlacademy-htmlcss:master' into master

Tigo929 committed:
Создаёт итоговуб базовую стилизацию

Tigo929 committed:
Merge branch 'master' of https://github.com/Tigo929/2388837-sedona-38

Tigo929 committed:
Добавляет ссылку

eepodoprigora commented:
Привет, 
1. Тегу body не хватает стиля font-weight;
2. Названия классов должны быть на англ, item_link-lupa , bread__item-strelka часть на русском, лучше назвать его item_link-search, bread__item-arrow;
3. Ссылка это строчный элемент, и ее ширина это ширина самого контента, тут не имеет смысла text-align: center;
4. В секции welcome тоже нечего ровнять по центру если ты поставил отступы. Либо можно без отступов слева и справа и тогда нужно центрирование
![image](https://github.com/htmlacademy-htmlcss/2388837-sedona-38/assets/89517901/d31c9755-2c2d-4153-baac-83c9063dc34f)
5. ПОИСК ГОСТИНИЦЫ В седоне см пункт 3

В каталоге
1.  немного не хватило картинки на всю секцию, в данном случай можно дать ей background-size: cover; Она размытая и не пострадает от растягивания
![image](https://github.com/htmlacademy-htmlcss/2388837-sedona-38/assets/89517901/24e5e294-92c3-4864-9418-1a738314eff1)

2. Переменные обычно задают 1 раз в root и потом используют по названия, пример ниже
![image](https://github.com/htmlacademy-htmlcss/2388837-sedona-38/assets/89517901/ecf82ad1-050b-4267-ad19-5b12db6466b5)

3. У form-contener по макету белый фон, а ты зачем его серым сделал)
4. Еще раз повтор, к строчным элементам не применятся text-align, лишнее свойство
![image](https://github.com/htmlacademy-htmlcss/2388837-sedona-38/assets/89517901/d69c0689-651d-491e-85ca-b1862d47bd80)

Вроде все, молодец!

Tigo929 commented:
Спасибо, за замечания.
Исправлю!

Tigo929 committed:
Делает глобальные сетки для страницы индекс

Tigo929 committed:
Исправляет ошибки по базовой стилизации

Исправил ошибки, которые выявил куратор.

Tigo929 committed:
Создает сетку для индекса

Tigo929 committed:
создаёт разметку для каталога

Tigo929 committed:
Исправляет ошибки

Исправил ошибки, которые были выявлены куратором.

eepodoprigora commented:
Привет, по данной теме все отлично теперь!
